packages/react-components/react-portal-compat/src/PortalCompatProvider.test.tsx

```diff
@@ -157,8 +157,8 @@ describe('PortalCompatProvider', () => {
     expect(result.current(element)).toBeInstanceOf(Function);
     expect(element.classList).toMatchInlineSnapshot(`
       DOMTokenList {
-        "0": "fui-FluentProvider2",
-        "1": "fui-FluentProvider1",
+        "0": "fui-FluentProviderr6",
+        "1": "fui-FluentProviderr5",
```

Comment:
We will need to implement a shared mock to `React.useId()` as this makes tests order dependent 😿